packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -337,6 +338,32 @@ svg.spectrum-Table-sortedIcon {
       }
     }
   }
+  &.spectrum-Table--isScrollingVertically {
```

Comment:
Not sure if there is a good way to resolve this btw (is it possible to differentiate between the two cases?):
If you have scroll bars showing "based on mouse or trackpad" you get a square corner as opposed to a rounded corner:
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/9661127/166839651-4df7f40c-7791-4083-835b-516c7677ba76.png">
vs old:
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/9661127/166839669-9b2315e6-91e0-4a2e-b851-65da6f3b9a95.png">

It looks good though if you have scrollbars on always and thus the scrollbar actually takes room to the right of the table:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/9661127/166839754-e42b86a3-c78b-4717-8d45-899d867ff678.png">

Reply:
Similar thing occurs for ListView. Also doesn't seem to be a problem in Safari, the scrollbar seems to always take room to the right of the table

Reply:
ok, just pushed a fix for these, I'll update the description test instructions to tell people to refresh the page